plugin.video.orftvthek/resources/lib/Addon.py

```diff
@@ -218,17 +216,18 @@ def run():
                     play_item.setProperty('inputstream', is_helper.inputstream_addon)
                     play_item.setProperty('inputstream.adaptive.manifest_type', input_stream_protocol)
                     play_item.setProperty('inputstream.adaptive.license_type', input_stream_drm_version)
-                    play_item.setProperty('inputstream.adaptive.license_key', lic_url + '||R{SSM}|')
+                    play_item.setProperty('inputstream.adaptive.license_key', lic_url + '|' + headers + '|R{SSM}|')
                 else:
                     streaming_url, play_item = scraper.liveStreamRestart(link, 'hls')
                     debugLog("Playing Non-DRM protected Restart Stream")
                     play_item.setProperty('inputstreamaddon', 'inputstream.adaptive')
                     play_item.setProperty('inputstream.adaptive.stream_headers', headers)
                     play_item.setProperty('inputstream.adaptive.manifest_type', 'hls')
                 debugLog("Restart Stream Url: %s; play_item: %s" % (streaming_url, play_item))
-                xbmcplugin.setResolvedUrl(pluginhandle, True, listitem=play_item)
-                listCallback(False, pluginhandle)
-                #xbmc.Player().play(streaming_url, play_item)
+                #This works on matrix. On Kodi <19 the stream wont play
+                #xbmcplugin.setResolvedUrl(pluginhandle, True, listitem=play_item)
+                #listCallback(False, pluginhandle)
+                xbmc.Player().play(streaming_url, play_item)
```

Comment:
It should work and the `xbmc.Player().play()` should not be used. You are missing a `play_item.setProperty("IsPlayable", "true")`.